cadence/gateway/cadence/grpc_service.py

```diff
@@ -22,25 +29,36 @@ def __init__(self, host: str = 'localhost', port: int = 7833, timeout: int = 300
         self.timeout = timeout
         self.channel = grpc.insecure_channel(
             '{}:{}'.format(host, port))
-
         # stubs
         self.domain = service_domain_pb2_grpc.DomainAPIStub(self.channel)
         self.service_workflow = service_workflow_pb2_grpc.WorkflowAPIStub(self.channel)
 
+    def close(self):
+        self.channel.close()
+
+    def set_next_timeout_cb(self, cb: Callable):
```

Comment:
Do you know what is this for?

Reply:
no, this is a copy paste from the tchannel version.